### PR TITLE
Add `eslint-plugin-vue-scoped-css`

### PR DESCRIPTION
**Pull request by silverwind**
Adds [eslint-plugin-vue-scoped-css](https://github.com/future-architect/eslint-plugin-vue-scoped-css) and fixes discovered issues which are:

- 1 unused selector
- 3 selectors with `.full.height` parent in a `<style scoped>` block so the rule could not find the parent. Move these into the unscoped block instead. They worked before and after.

### CODE REVIEW

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -676,22 +671,6 @@ export function initRepositoryActionView() {
 
 /* selectors here are intentionally exact to only match fullscreen */
 
-.full.height > .action-view-right {
```
Comment (wxiaoguang) [inline web_src/js/components/RepoActionView.vue:679]: I also ever questioned about this 😂

* https://github.com/go-gitea/gitea/pull/24876#discussion_r1209161891


Glad to see they are moved to a global scope.

File: web_src/js/components/RepoActionView.vue [hunk 2]
```diff
@@ -834,6 +813,22 @@ export function initRepositoryActionView() {
   margin-left: 10px;
 }
 
+.full.height > .action-view-right {
```
Comment (wxiaoguang) [inline web_src/js/components/RepoActionView.vue:816]: The comments like `/* selectors here are intentionally exact to only match fullscreen */` should also be moved here?

Comment (silverwind) [reply web_src/js/components/RepoActionView.vue:816]: done

**Review by wxiaoguang — APPROVED** [review]

File: web_src/js/components/RepoActionView.vue (hunk 1 above)
Comment (silverwind) [reply web_src/js/components/RepoActionView.vue:679]: Yes I was surprised it even worked, but apparently "scoped" style in Vue only means the final part of the selector needs to be local to the component, not the whole selector.

Pretty bad decision from Vue authors imho to not scope it completely to the component.

**Review by puni9869 — APPROVED** [review]

**Comment by puni9869** [comment]
let's ship this pr🚀

**Pull request merged**